### ISSUE TITLE
CSV grade uploader

### ISSUE THREAD

**Issue by cycomachead** [cs10/Alonzo#90]
If I drop a CSV link, Alonzo should parse the file and upload grades.

**Comment by aaaschmitt** [comment]
https://help.hipchat.com/forums/138883-suggestions-ideas/suggestions/7026468-allow-to-export-download-files-from-a-room

:(

**Comment by cycomachead** [comment]
But you can just grab message history and filter for files?
https://www.hipchat.com/docs/apiv2/method/view_recent_room_history

**Comment by aaaschmitt** [comment]
Ah okay. Have you started on this at all yet?

**Comment by cycomachead** [comment]
Not really. But this got me thinking to how you're updating the hipchat adapter. It could just expose the request interface.

**Comment by cycomachead** [comment]
In this case the grade uploader command would probably be upload grades to assignment ID using recent file and then Hubot can call the hipchat API.

**Comment by aaaschmitt** [comment]
yeah I was thinking about adding a function that allowed you to get the api id for the current room. Since this would just be exposing what I'm already doing for the sendFile stuff. Then you could use the hipchat api to make requests easily from hubot.
